src/pages/settings/email-settings/EmailSettings.tsx

```diff
@@ -94,6 +94,56 @@ export function EmailSettings() {
 
         <Divider />
 
+        <Element leftSide={t('email_provider')}>
+          <SelectField
+            value={company?.settings.email_sending_method}
+            onValueChange={(value) =>
+              handleChange('settings.email_sending_method', value)
+            }
+          >
+            <option defaultChecked value="default">
+              {t('default')}
+            </option>
+            <option value="gmail">{t('gmail')}</option>
+            <option value="microsoft">{t('microsoft')}</option>
+            <option value="client_postmark">{t('postmark')}</option>
+            <option value="client_mailgun">{t('mailgun')}</option>
```

Comment:
No need for translations for company names. They are same on all languges.

Reply:
Changed